wsl-pro-service/internal/system/system_test.go

```diff
@@ -390,23 +425,34 @@ func TestLandscapeEnable(t *testing.T) {
 	t.Parallel()
 
 	testCases := map[string]struct {
-		breakWriteConfig     bool
-		breakLandscapeConfig bool
-		breakWSLPath         bool
-		noLandscapeGroup     bool
+		landscapeConfigFile string
+
+		breakWriteConfigDir     bool
+		breakWriteConfig        bool
+		breakLandscapeConfigCmd bool
+		breakWSLPath            bool
+		breakWSLDistroName      bool
+		noLandscapeGroup        bool
 
 		wantErr bool
 	}{
-		"Success":                                                     {},
-		"Success overriding computer_title":                           {},
-		"Success overriding the SSL certficate path":                  {},
-		"Do not append wsl tag when config tag is provided":           {},
-		"Do not append wsl tag when config tag is provided but empty": {},
-
-		"Error when the file cannot be parsed":                   {wantErr: true},
-		"Error when the config file cannot be written":           {breakWriteConfig: true, wantErr: true},
-		"Error when the landscape-config command fails":          {breakLandscapeConfig: true, wantErr: true},
+		"Appends any required fields - no ssl":   {landscapeConfigFile: "minimal.conf"},
+		"Transform Windows SSL certificate path": {landscapeConfigFile: "windows_ssl_only.conf"},
+		"Refresh computer_title if changed":      {landscapeConfigFile: "old_computer_title.conf"},
+
+		"Regular with additional keys":            {landscapeConfigFile: "regular.conf"},
+		"Do not modify other sections and keys":   {landscapeConfigFile: "regular_with_extra_keys.conf"},
+		"Reformat landscape config to proper ini": {landscapeConfigFile: "regular_with_weird_format.conf"},
+
+		"Rerun landscape even without modifications": {landscapeConfigFile: "no_change_needed.conf"},
```

Comment:
I see what you mean about test case names storytelling. 👍🏽

Reply:
:book: